openshift/apimangement.yml

```diff
@@ -0,0 +1,39 @@
+apiVersion: batch/v1
```

Comment:
typo in this filename - should be `apimanagement.yml`

Reply:
fixed. thanks